torch/_export/serde/serialize.py

```diff
@@ -582,6 +585,14 @@ def serialize_optional_tensor_args(a):
             return Argument.create(as_memory_format=_TORCH_TO_SERIALIZE_MEMORY_FORMAT[arg])
         elif isinstance(arg, torch.layout):
             return Argument.create(as_layout=_TORCH_TO_SERIALIZE_LAYOUT[arg])
+        elif hasattr(type(arg), "__getstate__") and hasattr(type(arg), "__setstate__"):
```

Comment:
Should we check here that arg is indeed torch.classes? IIUC, this will only work for Torchbind classes right?